hs_core/models.py

```diff
@@ -4171,6 +4171,31 @@ def get_required_missing_elements(self):
 
         return missing_required_elements
 
+    def check_minimum_metadata_elements(self):
+        """Return a string indicating metadata elements that do not meet 'human in the loop' standards."""
+        missing = []
+        notification = ""
+
+        if not self.resource.raccess.public:
+            raise ValidationError("Minimum metadata should only be checked on public resources")
+
+        if len(self.title.value) < 30:
+            missing.append('the title must be at least 30 characters')
+        if len(self.description.abstract) < 150:
+            missing.append('your abstract must be at least 150 characters')
+        if self.subjects.count() < 3:
+            missing.append('you must include at least 3 keywords')
+
+        if missing:
+            notification = "Your resource doesn't meet the minimum metadata standards for publication: "
```

Comment:
Building a notification this way is a bit funky, but I didn't want to add these missing metadata elements into our content model and display with template (cm bloat). And Django Messages doesn't take formatting. Open to suggestions!